SimDataFormats/Associations/src/classes.h

```diff
@@ -39,12 +43,26 @@ namespace SimDataFormats_Associations {
 
     edm::Wrapper<hgcal::LayerClusterToSimTracksterAssociator> dummy11;
 
+    edm::Wrapper<reco::MtdRecoClusterToSimLayerClusterAssociator> dummy12;
+
     reco::VertexSimToRecoCollection vstrc;
     reco::VertexSimToRecoCollection::const_iterator vstrci;
     edm::Wrapper<reco::VertexSimToRecoCollection> wvstrc;
 
     reco::VertexRecoToSimCollection vrtsc;
     reco::VertexRecoToSimCollection::const_iterator vrtsci;
     edm::Wrapper<reco::VertexRecoToSimCollection> wvrtsci;
+
+    std::pair<FTLClusterRef, std::vector<MtdSimLayerClusterRef>> dummy13;
+    edm::Wrapper<std::pair<FTLClusterRef, std::vector<MtdSimLayerClusterRef>>> dummy14;
+    MtdRecoClusterToSimLayerClusterAssociationMap dummy15;
+    edm::Wrapper<MtdRecoClusterToSimLayerClusterAssociationMap> dummy16;
+
+    std::pair<MtdSimLayerClusterRef, std::vector<FTLClusterRef>> dummy17;
+    edm::Wrapper<std::pair<MtdSimLayerClusterRef, std::vector<FTLClusterRef>>> dummy18;
+    MtdSimLayerClusterToRecoClusterAssociationMap dummy19;
+    edm::Wrapper<MtdSimLayerClusterToRecoClusterAssociationMap> dummy20;
+
+    edm::Wrapper<reco::MtdSimLayerClusterToTPAssociator> dummy21;
```

Comment:
This entire `namespace SimDataFormats_Associations { ... }` has not been needed for a long time. Maybe we could use this PR to remove it?